addons/player-thumb/userscript.js

```diff
@@ -7,18 +7,25 @@ export default async function ({ addon, console }) {
   thumb.classList = "sa-project-thumb loading";
 
   const stageWrapper = await addon.tab.waitForElement('div[class*="stage-wrapper_stage-wrapper_"]');
+  addon.tab.redux.initialize();
+
+  // It's possible this runs after the project loads even without dynamic enable
+  if (addon.tab.redux.state?.scratchGui?.vmStatus?.started) return;
```

Comment:
```suggestion
  if (addon.tab.redux.state?.scratchGui?.projectState?.loadingState === "SHOWING_WITH_ID") return;
```
`vmStatus.started` is false if the project has been loaded but the green flag hasn't been clicked. To test this, you can add `await new Promise(resolve => setTimeout(resolve, 3000));` to the beginning of the userscript.